packages/create-next-app/helpers/examples.ts

```diff
@@ -69,3 +69,10 @@ export function downloadAndExtractExample(
     tar.extract({ cwd: root, strip: 3 }, [`next.js-canary/examples/${name}`])
   )
 }
+
+export async function listExamples(): Promise<any> {
+  const res = await got(
+    'https://api.github.com/repositories/70107786/contents/examples'
+  ).catch(e => e)
```

Comment:
When this fails we need to skip prompting the user to select an example. Currently, they'll get junk output.